### PR TITLE
[WIP] FCC Inclined ECal

### PR DESCRIPTION
**Pull request by vvolkl**
BEGINRELEASENOTES
- FCCee detector concept using hh-style LAr Calo, as per FCCSW sources

ENDRELEASENOTES
- [x] check compilation
- [ ] Add CaloData

### CODE REVIEW

**Comment by petricm** [comment]
The idea I proposed, to make as realistic as possible, is to take `FCCee_o1_v04` and replace `ECalBarrel_o2_v01_03.xml` and `ECalEndcap_o2_v01_03.xml` with that you have in this PR. The resulting model would be named `FCCee_o2_v04`.

File: FCCee/compact/FCCee_o2_v04/FCCee_o1_v04.xml [hunk 1]
```diff
@@ -0,0 +1,372 @@
+<lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
+    
+    <info name="FCCee_o1_v04"
```
Comment (petricm) [inline FCCee/compact/FCCee_o2_v04/FCCee_o1_v04.xml]: This file has to be renamed to FCCee_o1_v04 and this line need to be changed

File: FCCee/compact/FCCee_o2_v04/FCCee_o1_v04.xml [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+        title="FCCee detector model option 1 version 04 (update on MDI region)"
+	author="Yorgos Voutsinas, Anna Maria Kolano, Oleksandr Viazlo"
```
Comment (petricm) [inline FCCee/compact/FCCee_o2_v04/FCCee_o1_v04.xml]: some explanation here

File: FCCee/compact/FCCee_o2_v04/FCCee_o1_v04.xml [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+    <include ref="HOMAbsorber_v00.xml"/>
+    <include ref="BeamInstrumentation_o3_v02_fitShield.xml"/>
+
+    <include ref="LumiCal_o3_v02_03.xml"/>
+
+    <include ref="Vertex_o4_v05.xml"/>
+    <include ref="InnerTracker_o2_v06_02.xml"/>
+    <include ref="OuterTracker_o2_v06_02.xml"/>
+
+    <include ref="ECalBarrel_o2_v01_03.xml"/>
```
Comment (petricm) [inline FCCee/compact/FCCee_o2_v04/FCCee_o1_v04.xml]: Here you need to include `ECalLArBarrel_o2_v01_01.xml` and remove `ECalBarrel_o2_v01_03.xml`

File: FCCee/compact/FCCee_o2_v04/FCCee_o1_v04.xml [hunk 4]
```diff
@@ -0,0 +1,372 @@
...
+    <include ref="ECalEndcap_o2_v01_03.xml"/>
```
Comment (petricm) [inline FCCee/compact/FCCee_o2_v04/FCCee_o1_v04.xml]: What are your plans for the Endcap?

Comment (vvolkl) [reply FCCee/compact/FCCee_o2_v04/FCCee_o1_v04.xml]: Yes, I have a model for the endcaps as well. I first wanted to check if the simulation with the barrel is running, but I can try to include them now.

**Pull request closed without merging**